src/App.js

```diff
@@ -2,32 +2,40 @@ import React, { Component } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+import { Search } from './components/Search/Search';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  };
+
+  handleChange = (event) => {
+    const { name, value } = event.target;
+
+    this.setState({ [name]: value.toLocaleLowerCase() });
```

Comment:
```suggestion
    this.setState({ query: value.toLocaleLowerCase() });
```
у тебя всего 1 инпут, поэтому можно записывать явно в query